src/datalink/bpf.rs

```diff
@@ -27,11 +26,12 @@ use util::NetworkInterface;
 
 // NOTE buffer must be word aligned.
 #[inline]
-pub fn datalink_channel(network_interface: &NetworkInterface,
-                        write_buffer_size: usize,
-                        read_buffer_size: usize,
-                        channel_type: DataLinkChannelType)
-    -> io::Result<(Box<EthernetDataLinkSender>, Box<EthernetDataLinkReceiver>)> {
+pub fn datalink_channel
+    (network_interface: &NetworkInterface,
```

Comment:
I'm sure there's an option to stop it doing this in there - could you investigate? There's already a rustfmt configuration file in the repo.

I would love to start using rustfmt for everything, but unfortunately it's not currently possible (correct me if I'm wrong/this has changed) to exclude areas. Specifically, see https://github.com/libpnet/libpnet/blob/f08e08e76e6dc7498171982f58caa7b2c1270b89/src/packet/ipv4.rs.in#L263-L272 - last time I checked, rustfmt absolutely destroyed the formatting on that array literal, making it completely illegible.

If it doesn't do that any more, I'd be happy to start using it, and perhaps integrating it with the CI.